Resources/Prototypes/Research/experimental.yml

```diff
@@ -157,3 +157,15 @@
   recipeUnlocks:
     - WeaponForceGun
     - WeaponTetherGun
+
+- type: technology
+  id: SpeedTech
+  name: research-technology-speed-tech
+  icon:
+    sprite: Clothing/Shoes/Boots/speedboots.rsi
+    state: icon
+  discipline: Experimental
+  tier: 3
```

Comment:
no point that a 10% speed boost should be tier 3. Experimental already has too many t3 techs already